quickget

```diff
@@ -2316,6 +2316,9 @@ function get_opensuse() {
     elif [ "${RELEASE}" == 15.0 ] || [ "${RELEASE}" == 15.1 ]; then
         ISO="openSUSE-Leap-${RELEASE}-DVD-x86_64.iso"
         URL="https://download.opensuse.org/distribution/leap/${RELEASE}/iso"
+    elif [ "${RELEASE}" == 16.0 ]; then
+        ISO="Leap-${RELEASE}-online-installer-x86_64.install.iso"
```

Comment:
```suggestion
        ISO="Leap-${RELEASE}-offline-installer-x86_64.install.iso"
```
I think we should continue to grab the full offline ISO by default.  Enabling the smaller online version would be better, in my view, if presented as an optional ${EDITION}